release_notes/osd_latest_product_updates.adoc

```diff
@@ -0,0 +1,517 @@
+- Setting the `*TERM*` environment variable so common shell actions like `clear` behave the way you expect
+- Better support for multi-container pods
+
+.Terminal
+image::ocp33-terminal.png["Terminal"]
+
+[[osd-web-console-image-details]]
+==== Image Details
+
+Before the latest update, there was no information in the web console about
+the images in your image streams, aside from the SHAs. This made it difficult to
+know the specifics of how your image was defined unless you used the CLI. Now,
+for any image stream tag you can see the metadata, cofiguration, and layers.
+
+.Image Stream Tag Details
+image::ocp33-imagedetails.png["Image Stream Tag Details"]
+
+.Image Stream Tag Configuration
+image::ocp33-imagedetails2.png["Image Stream Tag Configuration"]
+
+[[osd-networking]]
+=== Networking
+
+This release adds the following improvements to networking components.
+
+[[osd-ab-service-annotation]]
+==== A/B Service Annotation
+
+The latest update adds service lists to routes, making it easier to perform
+A/B testing. Each route can now have multiple services assigned to it, and those
+services can come from different applications or pods.
+
+New automation enables HAProxy to be able to read weight annotations on the
+route for the services. This enables developers to declare traffic flow (for
+example, 70% to application A and 30% to application B) using the CLI or web
+console.
+
+[NOTE]
+====
+See xref:osd-web-console-ab-routing[New Features and Enhancements: Web Console] for
+more details on the new A/B routing user interface.
+====
+
+See xref:../dev_guide/routes.adoc#routes-load-balancing-for-AB-testing[Load Balancing for A/B Testing] for more details.
+
+[[osd-security]]
+=== Security
+
+This release adds the following improvements to cluster security.
+
+[[osd-scc-profiles-seccomp]]
+==== SCC Profiles for seccomp
+
+The *seccomp* feature in Red Hat Enterprise Linux (RHEL) has been enabled for docker 1.10 or higher. This feature allows containers to define interactions with the kernel using *syscall* filtering. This reduces the risk of a malicious container exploiting a kernel vulnerability, thereby reducing the guest attack surface.
+
+{product-title} adds the ability to create *seccomp* policies with security
+context constraints (SCCs). This allows Red Hat Operations to set SCC
+policies on developers that imposes a filter on their containers for Linux-level
+system calls.
+
+See the xref:../architecture/additional_concepts/authorization.adoc#authorization-seccomp[Authorization] concept for more details.
+
+[[osd-cluster-longevity]]
+=== Cluster Longevity
+
+This release adds the following improvements to cluster longevity.
+
+[[osd-scale]]
+==== Scale
+
+1000 nodes per cluster at 250 pods per node (with a
+recommendation of 10 pods per hyper-threaded core) are now supported. See
+link:https://docs.openshift.com/container-platform/3.3/install_config/install/planning.html#sizing[Sizing Considerations] for
```

Comment:
Could make it clearer here they're going to be taken to the OCP docs. E.g., "See the OpenShift Container Platform [Sizing Considerations] for more details."